site/src/App.tsx

```diff
@@ -60,10 +74,10 @@ export const AppProviders: FC<AppProvidersProps> = ({
 
 export const App: FC = () => {
   return (
-    <AppProviders>
-      <ErrorBoundary>
+    <ErrorBoundary>
+      <AppProviders>
         <RouterProvider router={router} />
-      </ErrorBoundary>
-    </AppProviders>
+      </AppProviders>
+    </ErrorBoundary>
```

Comment:
Important – before, if any of the sub-components in `AppProviders` threw an error, we had nothing to catch it. Moved `ErrorBoundary` to be the top-most component